runtime/ftplugin/query.lua

```diff
@@ -14,8 +14,8 @@ vim.treesitter.start()
 -- set omnifunc
 vim.bo.omnifunc = 'v:lua.vim.treesitter.query.omnifunc'
 
--- recognize @foo.bar as a single word
-vim.opt.iskeyword:append('.')
+-- recognize @foo.bar as a single word in the query previewer
```

Comment:
this additional comment isn't needed and actually wrong
this changes the settings for the `query` filetype so in all query files. And there is no need to write this because it's obvious since it's in the ftplugin